be/src/exprs/runtime_filter_slots.h

```diff
@@ -220,7 +220,11 @@ class RuntimeFilterSlotsBase {
     Status publish() {
         for (auto& pair : _runtime_filters) {
             for (auto filter : pair.second) {
-                RETURN_IF_ERROR(filter->publish());
+                auto state = filter->publish();
+                if (!state) {
+                    // TODO: solve publish failed when scan not schedured
+                    LOG(WARNING) << "filter publish failed, reason=" << state.to_string();
```

Comment:
If state is not ok, why not return error status?

Reply:
Even if rf publish failed, the query should execute normally.
I changed this behavior in #20593 to cause the workflow to be blocked, now change it back